.template/README.md

```diff
@@ -21,48 +21,52 @@
 
 TODO: A summary of the application should be written here.
 
-More information can be found in the [specification](./project/SPECIFICATION.md).
+More information about the contract can be found [here](./project/SPECIFICATION.md) and for the user interface [here](./ui/SPECIFICATION.md).
 
-## Project Structure
+## Project structure
 
 The project consists of a smart contract.
 
 <!--Only show most important files e.g. script to run, build etc.-->
 
-```
-Template/
-├── project/
-|   └── template-contract/
-|       ├── src/main.sw
-|       └── tests/harness.rs
-├── README.md
-└── SPECIFICATION.md
+```sh
+Template
+├── project
+│   ├── contracts
+│   │   └── template-contract
+│   │       ├── src/main.sw
+│   │       └── tests/harness.rs
+│   ├── libraries
+│   ├── predicates
+│   ├── scripts
+│   ├── README.md
+│   └── SPECIFICATION.md
+├── ui
+│   ├── README.md
+│   └── SPECIFICATION.md
+└── README.md
 ```
 
 ## Running the project
 
-### User Interface
+### User interface
 
-TODO: UI does not currently exist
+TODO: The user interface does not currently exist therefore its [README.md](ui/README.md) and [SPECIFICATION.md](ui/SPECIFICATION.md) are empty.
```

Comment:
When one creates an application from this template, the contracts do not exist either. Why have a TODO for UI when everything is TODO?

Reply:
It's a template so there are TODOs. 

The UI has no content at all therefore this comment exists. The contracts have a template to be filled in therefore they are not mentioned.